config/samples/capi/capi-controlplane-hetzner.yaml

```diff
@@ -24,7 +24,7 @@ kind: K0smotronControlPlane # This would somehow map to k0smotron
 metadata:
   name: cp-test
 spec:
-  k0sVersion: v1.27.2-k0s.0
+  version: v1.27.2-k0s.0
```

Comment:
Should this be `v1.27.2+k0s.0`?

Reply:
(it's also like this in the couple of examples below this)

Reply:
No, here it is the docker image tag, and we use `-` there.